nbopen.py

```diff
@@ -47,7 +49,11 @@ def nbopen(filename, profile='default'):
             nbdir = os.path.dirname(filename)
 
         print("Starting new server")
-        notebookapp.launch_new_instance(file_to_run=os.path.abspath(filename),
+        path = os.path.relpath(filename, start=nbdir)
+        notebook_url = url_path_join('notebooks', url_escape(path))
+        if query:
+            notebook_url = notebook_url + query
+        notebookapp.launch_new_instance(default_url=notebook_url,
```

Comment:
`default_url` will, for the lifetime of the notebook server it starts, redirect requests for `/` to `default_url`. I'd rather not change the behaviour of the notebook server like that just to get it to open the right page once.